modin/pandas/test/test_api.py

```diff
@@ -33,11 +33,9 @@ def test_top_level_api_equality():
         "core",
         "compat",
         "util",
-        "offsets",
         "datetime",
```

Comment:
Datetime and np are not in the namespace in main anymore

Reply:
datetime isn't in `dir(pandas)`, but I can do `from pandas import datetime`. Not sure how that works. I removed `np`.

Reply:
> but I can do from pandas import datetime

sure you're using main?

Reply:
I just tried main-- I get `ImportError: cannot import name 'get_supported_reso' from 'pandas._libs.tslibs.dtypes'`. But Modin CI isn't using main pandas, so we can't change this yet.

Reply:
you likely need to re-build